### ISSUE TITLE
NNCF skips insert position for FakeQuanizer

### ISSUE THREAD

**Issue by kshpv** [openvinotoolkit/nncf#188]
Comparing models graphs of compressed ssd300 by NNCF and POT it was noticed that they are not the same however one of the requirements to NNCF is to build the POT-like graph. Moreover, it seems that NNCF didn't put several FakeQuanztizers where I expected. There are two images below:
1) There is illustrated the start point of the NNCF-compressed model graph. I suppose that there should **be one more FakeQuantizer** (I underlined a location by a red pen)
2) There is one more place where NNCF differs with POT graphs. The same model's location are underlined by red pen. **NNCF didn't put any FakeQuantizer** there, while POT did. **The left image corresponds to POT and right to NNCF**

If you would like to have a look at the full model's graphs. Please contact me, I will share them here or privately.

![ssd300_int8_LI (3)](https://user-images.githubusercontent.com/32935044/95868842-5c8bd480-0d73-11eb-997f-070e4de9dc36.jpg)
![InkedPOT_NNCF_graphs_ssd300_int8_LI (3)](https://user-images.githubusercontent.com/32935044/95868921-77f6df80-0d73-11eb-93ab-4db14175622e.jpg)
_Left image - POT; right image - NNCF_

**Comment by vshampor** [comment]
@aleksei-kashapov could you retry with latest NNCF version?

**Comment by aleksei-kashapov** [comment]
@vshampor 
The issue with lacking of FQ on the input tensor was resolved.
As you asked I quantized the same model by NNCF 1.5. with default quantization mode. As a result the problem with the pattern (Power, ReduceSum, Power, Add, Power) still remains
![Screenshot 2020-11-12 152822 (2)](https://user-images.githubusercontent.com/32935044/98940360-eb6b4880-24fb-11eb-8e2f-b1f79d37a67a.png)

**Comment by vshampor** [comment]
@aleksei-kashapov which operation in PyTorch does this problematic subgraph correspond to?

**Comment by aleksei-kashapov** [comment]
@vshampor 
```
class L2Norm(nn.Module):

    def __init__(self, n_dims, scale=20., eps=1e-10):
        super(L2Norm, self).__init__()
        self.n_dims = n_dims
        self.weight = nn.Parameter(torch.Tensor(self.n_dims))
        self.eps = eps
        self.scale = scale

    def forward(self, x):
        # normalization layer convert to FP32 in FP16 training
        x_float = x.float()
        norm = x_float.pow(2).sum(1, keepdim=True).sqrt() + self.eps
        return (self.weight[None, :, None, None].float().expand_as(x_float) *
                x_float / norm).type_as(x)
```

**Comment by vshampor** [comment]
@aleksei-kashapov did `@register_module` approach work in this case?

**Comment by vshampor** [comment]
@aleksei-kashapov still waiting for you to verify.

**Comment by vshampor** [comment]
@aleksei-kashapov did you have a chance to verify it?

**Comment by aleksei-kashapov** [comment]
@vshampor I am really sorry. I apologize, I will check it out soon

**Comment by aleksei-kashapov** [comment]
should be resolved by https://github.com/openvinotoolkit/nncf/pull/637

**Comment by kshpv** [comment]
Should be fixed with https://github.com/openvinotoolkit/nncf/pull/984

**Comment by kshpv** [comment]
The issue is resolved by #984
